client/client.go

```diff
@@ -595,7 +595,8 @@ type TSFuture interface {
 func (req *tsoRequest) Wait() (physical int64, logical int64, err error) {
 	// If tso command duration is observed very high, the reason could be it
 	// takes too long for Wait() be called.
-	cmdDuration.WithLabelValues("tso_async_wait").Observe(time.Since(req.start).Seconds())
+	now := time.Now()
+	cmdDuration.WithLabelValues("tso_async_wait").Observe(now.Sub(req.start).Seconds())
```

Comment:
@tiancaiamao  by the wait, plz help change this to

```
var cmdWaitDuration = cmdDuration.WithLabelValues("tso_async_wait");

....

cmdWaitDuration.Observe(now.Sub(req.start).Seconds())
```

Reply:
I think it is not a related change. We could modify them in another PR.

Reply:
Done